### DISCUSSION TITLE
translation

### DISCUSSION THREAD

**Discussion by iFallenHunt** [GrapesJS/grapesjs#4365] [Q&A]
Hello, I am starting to use github and would like to contribute to some projects, I have translated readme.md into Brazilian Portuguese, I would like to know where I can send it

**Comment by ronaldohoch** [comment]
Olá!
Acho que você pode criar uma pasta com o nome locale e vê se o Arthur aceita :D

**Reply by iFallenHunt** [reply]
Ufa, que alivio ver mais brasileiros aqui haha
sou novo nos commit e to realizando algumas traduções pra ir treinando mais até me sentir confiante para trabalhar nos códigos.

vou seguir a sua dica, ja estou com tudo pronto aqui, vlw

**Reply by ronaldohoch** [reply]
Eu vou te mandar alguns exemplos antes de fazer aí

**Reply by ronaldohoch** [reply]
Esse aqui eles colocaram no original um botão onde leva para as pastas traduzidas.
https://github.com/kamranahmedse/developer-roadmap
https://github.com/kamranahmedse/developer-roadmap/tree/0471d44c8fae58b6a36a7c57bba12253916d0249/translations

Ta, só achei esse na real :/ Mas já vi outros nesse estilo

**Reply by iFallenHunt** [reply]
legal, montei algo nesse sentido, vou enviar, espero conseguir contribuir
